packages/ilib-lint/src/Project.js

```diff
@@ -63,10 +63,11 @@ const unknownFileTypeDefinition = {
  * @private
  * @param {Object} instance the instance to check
  * @param {String} methodName the name of the method to check
+ * @param {Formatter} formatter the formatter class
```

Comment:
Formatter as the type means an instance of the Formatter class. What you mean is the Formatter class itself. Also, this function works no matter what class this is. Instead of calling it formatter, call it something like parentClass:

```suggestion
 * @param {Class} parentClass the parent class of the instance or one of its ancestors
```

Reply:
yes, I updated it